src/aat/resources/features/F-047/F-047.feature

```diff
@@ -0,0 +1,108 @@
+    Given a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [does not provide authorised access to the operation]
+    And it is submitted to call the [Get case ids] operation of [CCD Data Store]
+    Then a negative response is received
+    And the response [contains an HTTP-403 Forbidden]
+    And the response has all other details as expected
+
+  @S-570 @Ignore # Response code mismatch, expected: 400, actual: 500
+  Scenario: must return negative response HTTP-400 when request contains a malformed user ID
+    Given a user with [an inactive profile in CCD]
+    When  a request is prepared with appropriate values
+    And   the request [contains a malformed user ID]
+    And   it is submitted to call the [Get case ids] operation of [CCD Data Store]
+    Then  a negative response is received
+    And   the response [code is HTTP-400]
+    And   the response has all other details as expected
+
+  @S-571 @Ignore # Response code mismatch, expected: 400, actual: 500
+  Scenario: must return negative response HTTP-400 when request contains a malformed case type ID
+    Given a user with [an active profile in CCD]
+    When  a request is prepared with appropriate values
+    And   the request [contains a malformed case type ID]
+    And   it is submitted to call the [Get case ids] operation of [CCD Data Store]
+    Then  a negative response is received
+    And   the response [code is HTTP-400]
+    And   the response has all other details as expected
+
+  @S-572 @Ignore # Response code mismatch, expected: 400, actual: 403
+  Scenario: must return negative response HTTP-400 when request contains a malformed jurisdiction ID
+    Given a user with [an active profile in CCD]
+    When  a request is prepared with appropriate values
+    And   the request [contains a malformed jurisdiction ID]
+    And   it is submitted to call the [Get case ids] operation of [CCD Data Store]
+    Then  a negative response is received
+    And   the response [code is HTTP-400]
+    And   the response has all other details as expected
+
+  @S-573 @Ignore # Response code mismatch, expected: 400, actual: 403
+  Scenario: must return negative response HTTP-400 when request contains a non-existing jurisdiction ID
+    Given a user with [an active profile in CCD]
+    When  a request is prepared with appropriate values
+    And   the request [contains a non-existing jurisdiction ID]
+    And   it is submitted to call the [Get case ids] operation of [CCD Data Store]
+    Then  a negative response is received
+    And   the response [code is HTTP-400]
+    And   the response has all other details as expected
+
+  @S-574 @Ignore #Response code mismatch, expected: 400, actual: 200"
+  Scenario: must return negative response HTTP-404 when request contains a non-existing case type ID
+    Given a user with [an active profile in CCD]
```

Comment:
Please update the defect number.
Also please update comment : expected code is 404.
Also add this scenario in Jira RDM-6387

Reply:
done